### ISSUE TITLE
Error on COUNT SQL Function from Java Code

### ISSUE THREAD

**Issue by vsspt** [orientechnologies/orientdb#7246]
## OrientDB Version, operating system, or hardware.
v2.2.17

## Operating System
- [x ] Windows 8

## JVM
- 1.8.0_101-b13

## Frameworks/Libs
- Spring Boot 1.5.2 

## Expected behavior and actual behavior
When executing a COUNT(1) Query, the result from OrientDB console is correct, but through code is not.

## Steps to reproduce the problem

Please download this test project here: https://meocloud.pt/link/25256186-4f81-49d5-a5f3-dafd8a78d87c/2-2-17.zip/

In order to use the project, you will need to change the plocal url in the file: srt\test\testContext.xml.

Basically, this is a simple Spring application that executes a simple Count Query:

```
private final String dmlSelect = "SELECT COUNT(1) from TABLE_CONFIG WHERE STATUS = 1";

public Integer select() {
	return getJdbcTemplate().queryForObject(dmlSelect, Integer.class);
}
```

- When executing from Code, the result of the Query is 0 (INCORRECT)
- When executing from OrientDB console, the result is 1 (CORRECT)
